### ISSUE TITLE
boundaryCanonical field completeness

### ISSUE THREAD

**Issue by leeberryman** [wmgeolab/geoBoundaries#1992]
As I have been trying to leverage the boundaryCanonical field on each of the layers from the ISOs in geoboundaries, I have found it to be incomplete.  I have a running spreadsheet where I'm looking this up and populating it.  Would there be an easy way to contribute this back to the project or would it require touching each of the source files and its metadata fields?

**Comment by DanRunfola** [comment]
Yes - we have been updating boundaryCanonical slowly but it is definitely incomplete!

There isn't an "easy" way, but if you provide a spreadsheet I could ask one of our members here to use that to update the metadata with the information you collect.  It does require touching all of the source zip files on the back-end.

**Comment by leeberryman** [comment]
Ok thanks @DanRunfola I'll share the spreadsheet at the end of the week when I'm done.

**Comment by ewrayjohnson** [comment]
Not only is _boundaryCanonical_ incomplete, but: 
1. non-standard values are not overridden
2. the value are plural when they should be singular IMHO;
3. _boundaryCanonical_ seems to have dual purpose: a) an alternate name(s) especially when for example special Unicode characters are used (e.g. AFG-ADM0 has the value `"\u0627\u0641\u063a\u0627\u0646\u0633\u062a\u0627\u0646\u200e , Af\u0121\u0101nist\u0101n"`); and b) a category name of the features (USA-ADM2 has the value `"Counties"`)

1. For example, the U.S. state of Louisiana is divided into 64 parishes ([French](https://en.wikipedia.org/wiki/French_language): paroisses, [Spanish](https://en.wikipedia.org/wiki/Spanish_language): parroquias) in the same manner that [Alaska](https://en.wikipedia.org/wiki/Alaska) is divided into [boroughs](https://en.wikipedia.org/wiki/List_of_boroughs_and_census_areas_in_Alaska), and 48 other states are divided into [counties](https://en.wikipedia.org/wiki/County_(United_States)).  
2. The USA-ADM2 has the value `"Counties"` and IMHO should instead be `"County"`.

**Comment by DanRunfola** [comment]
@ewrayjohnson Yes - to respond to 1 and 3, we definitely do not guarantee the consistency of the field right now (triply so as it isn't even populated for all cases yet).  I do think we eventually need to have a more detailed discussion around how we want to define the column, but we have been prioritizing geometric issues over meta issues for now.  Quite open to alternative schema designs, but I don't want to delay current collection efforts (perfect being the enemy of the good, and all).

-Dan

**Comment by christillotson** [comment]
Discussed 9-30-2025 to leave this issue stand for now because we are still prioritizing geometric issues over meta issues.
